### DISCUSSION TITLE
What kind of server needs?

### DISCUSSION THREAD

**Discussion by okntubek** [infiniflow/ragflow#3923] [General]
Hello,

I tried to install RagFlow to server but I got some warnings like at below. there are some server cpu limiations. What kind of server you success to install? Do you suggest me any cloud service that suppling that? Thank you. 
 
It is not ARM or MAC. Ubuntu 22.04 LTS , 24 CPU, 32 GB RAM. Intel(R) Xeon(R) CPU E3-1240 V2 @ 3.40GHz

warnings.warn(

./entrypoint.sh: line 35:    73 Illegal instruction     (core dumped) $PY api/ragflow_server.py
2024-12-09 01:02:21,908 WARNING  75 /ragflow/.venv/lib/python3.10/site-packages/polars/_cpu_check.py:259: RuntimeWarning: Missing required CPU features.

The following required CPU features were not detected:
    avx2, fma, bmi1, bmi2, lzcnt, movbe
Continuing to use this version of Polars on this processor will likely result in a crash.
Install the `polars-lts-cpu` package instead of `polars` to run Polars with better compatibility.

Hint: If you are on an Apple ARM machine (e.g. M1) this is likely due to running Python under Rosetta.
It is recommended to install a native version of Python that does not run under Rosetta x86-64 emulation.

If you believe this warning to be a false positive, you can set the `POLARS_SKIP_CPU_CHECK` environment variable to bypass this check.

**Comment by moyang-creator** [comment]
I've encountered the same problem. Is there a solution yet? My CPU is Intel(R) Xeon(R)  E5-1650V2 @ 3.50GHz
